documentation/book/appendix_metrics.adoc

```diff
@@ -52,19 +52,33 @@ The following resources are defined:
 
 ==== Deploying on {OpenShiftName}
 
-To deploy all these resources you can run the following.  Note that this file creates a `ClusterRoleBinding` in the `myproject` namespace.  If you're not using this namespace then download the resource file locally and update it.
+The provided YAML file, with all the Prometheus related resources, creates a `ClusterRoleBinding` in the `myproject` namespace.
+If you're not using this namespace then download the resource file locally and update it as follow.
+
+[source,shell,subs=attributes+]
+curl -o prometheus.yaml https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/kubernetes.yaml
```

Comment:
Can't we somehow pipe it from `curl`, through `sed` into `oc` to keep this as single command?

Reply:
I have piped curl and sed but I would leave the apply as a separate command.